### PR TITLE
added e2e test case for tree

### PR DESCRIPTION
**Pull request by NileshPatel17**
added e2e test cases for #793

### CODE REVIEW

File: apps/sensenet/cypress/integration/tree/tree.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+const items = ['Calendar', 'Document library', 'Groups', 'Links', 'Memos', 'Tasks', 'Image Library']
+
+describe('Tree', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+  it('Tree item should open with the list of its children.', () => {
+    cy.get('[data-test="Content"]').click()
+    cy.get('[data-test="menu-item-Sample workspace"]')
```
Comment (pusztaienike) [inline apps/sensenet/cypress/integration/tree/tree.spec.ts:12]:
```suggestion
    cy.get('[data-test="menu-item-IT Workspace"]')
```

File: apps/sensenet/cypress/integration/tree/tree.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      .click()
+      .then(() => {
+        items.forEach(($el) => {
+          cy.get(`[data-test="menu-item-${$el}"]`).should('be.visible')
+        })
+      })
+  })
+  it('Click on the Tree item when it is open should close it and make its children invisible.', () => {
+    cy.get('[data-test="Content"]').click()
+    cy.get('[data-test="menu-item-Sample workspace"]')
```
Comment (pusztaienike) [inline apps/sensenet/cypress/integration/tree/tree.spec.ts:22]:
```suggestion
    cy.get('[data-test="menu-item-IT Workspace"]')
```

File: apps/sensenet/cypress/integration/tree/tree.spec.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      .rightclick()
```
Comment (pusztaienike) [inline apps/sensenet/cypress/integration/tree/tree.spec.ts:23]:
```suggestion
      .click()
```

File: apps/sensenet/cypress/integration/tree/tree.spec.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      .then(() => {
+        cy.get('[data-test="menu-item-Sample workspace"]')
```
Comment (pusztaienike) [inline apps/sensenet/cypress/integration/tree/tree.spec.ts:25]:
```suggestion
        cy.get('[data-test="menu-item-IT Workspace"]')
```

**Review by pusztaienike — COMMENTED** [review]
We have a dedicated environment for end2end tests where there is a content with the name of 'IT Workspace' - so it is okay :)

**Comment by NileshPatel17** [comment]
@pusztaienike changes are done as requested.

**Comment by herflis** [comment]
Thank you @NileshPatel17

**Pull request merged**